buildscript.txt

```diff
@@ -1,10 +1,10 @@
 real-time
---public-repo=True
---language=coffeescript
---env-add=
---node-version=10.21.0
 --acceptance-creds=None
---dependencies=redis
+--dependencies=None
```

Comment:
Please restore the old buildscript.txt file. The current revision is missing `public-repo=True` and `dependencies=redis`. CI is marked as success but in reality it failed.